### ISSUE TITLE
Remove duplicate Device Identifiers in Grower Detail

### ISSUE THREAD

**Issue by nmcharlton** [Greenstand/treetracker-admin-client#146]
It's possible for a planter to have multiple registrations on the same device, for example if the Treetracker app has been reinstalled.
We need to remove duplicate _Device Identifiers_ from the list in the Planter Detail dialog.

![Screenshot 2021-08-14 at 14 52 38](https://user-images.githubusercontent.com/5558838/129448627-c949f1f3-847d-40ce-a611-4670d33100c1.png)

**Comment by nmcharlton** [comment]
@panzaverdeok Do you want to take a look at this issue?

**Comment by nmcharlton** [comment]
Hi @panzaverdeok 

I have limited internet access at the moment, but the simplest way to remove duplicates from an array is to cast it to a `Set`. Perhaps something like:
```
setDeviceIdentifiers(
  [...Set(sortedRegistrations
    .map((reg) => reg.device_identifier)
    .filter((id) => id),
  )]
);
```
If you were to do it like this, I would recommend adding a comment to the code to explain why.

**Comment by nmcharlton** [comment]
Hi @panzaverdeok. Did you end up making any progress on this issue?

**Comment by nmcharlton** [comment]
Resolved by #279
